src/features/locators/utils/locatorOutput.ts

```diff
@@ -1,7 +1,8 @@
 import { LocatorType } from "../../../common/types/common";
 import { LocatorValue } from "../types/locator.types";
+import { CALCULATING } from "./constants";
 
 export const getLocator = (locatorValue: LocatorValue, locatorType: LocatorType = LocatorType.xPath) => {
   if (locatorType !== LocatorType.cssSelector) return locatorValue.xPath;
-  return locatorValue.cssSelector;
+  return locatorValue.cssSelector || CALCULATING;
```

Comment:
Can we have a situation when xPath also will be calculated?

Reply:
No, we can't. CALCULATING is shown when Css is undefined. xPath can't be undefined, at least it will  be fullXpath value